core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.spec.ts

```diff
@@ -49,6 +68,7 @@ describe('ExperimentsListComponent', () => {
     const createComponent = createComponentFactory({
         component: DotExperimentsListComponent,
         componentProviders: [provideComponentStore(DotExperimentsListStore)],
+        imports: [DotAddToBundleModule, HttpClientTestingModule],
```

Comment:
Try the same as the prev. Mock the module, and remove the extras providers added. And I think `HttpClientTestingModule` is not necessary.